pkg/sql/schemachanger/scexec/dependencies.go

```diff
@@ -96,6 +97,13 @@ type Catalog interface {
 	Reset(ctx context.Context) error
 }
 
+// RelationHelpers utilities that aid in the creation of
+// relations.
+type RelationHelpers interface {
+	// InitializeSequence initializes the initial value for a sequence.
+	InitializeSequence(id descpb.ID, startVal int64)
+}
```

Comment:
This is fine, though FYI I wouldn't have minded simply adding the `InitializeSequence` method to `scexec.Catalog`, because I'm not sure how much this interface will grow in the future.